### PR TITLE
RTP6b

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Spec/RealtimeClientPresence.swift [hunk 1]
```diff
@@ -77,6 +77,36 @@ class RealtimeClientPresence: QuickSpec {
...
+                    waitUntil(timeout: testTimeout) { done in
+                        channel1.presence.subscribe(.Update) { member in
+                            expect(member.action).to(equal(ARTPresenceAction.Update))
+                            expect(member.clientId).to(equal("john"))
+                            expect(member.data as? NSObject).to(equal("away"))
+                        }
+                        channel2.presence.enterClient("john", data: "online")
+                        channel2.presence.updateClient("john", data: "away")
+                        channel2.presence.leaveClient("john", data: nil)
+                        done()
```
Comment (mattheworiordan) [inline Spec/RealtimeClientPresence.swift]: You are not waiting to check that the update event was received by the subscribe callback, this test could complete before that assertion.

Comment (tcard) [inline Spec/RealtimeClientPresence.swift]: Agreed. Plus, you have to make sure that no other event is recorded. I suspect this won't work provided that #319 doesn't work, so I don't think this test would pass.

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: > Plus, you have to make sure that no other event is recorded.

@tcard Can you clarify, please.

Comment (tcard) [inline Spec/RealtimeClientPresence.swift]: Nothing, actually, I somehow missed [this line](https://github.com/ably/ably-ios/pull/320/commits/dc09a31b1dbf0df065bbbf3e0100d9a4d6fdbb5f#diff-b7ac3256e97dbdaeec04a488430a2669R97)...

Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: Done dc09a31.

**Comment by mattheworiordan** [comment]
LGTM

**Comment by tcard** [comment]
LGTM

**Pull request merged**